kartothek/io/dask/_shuffle.py

```diff
@@ -35,7 +41,7 @@ def _hash_bucket(df: pd.DataFrame, subset: Optional[Sequence[str]], num_buckets:
 def shuffle_store_dask_partitions(
     ddf: dd.DataFrame,
     table: str,
-    secondary_indices: Optional[Union[str, Sequence[str]]],
+    secondary_indices: Optional[Union[Literal[False], Sequence[str]]],
```

Comment:
This argument is merely passed through to `_unpack_store_partition`, whose signature is a bit different. Please make the type annotations consistent.

Reply:
- [ ] check annotations again